index.html

```diff
@@ -1,29 +1,138 @@
+
+<body>
+    <div class="main-Container">
+        <header class="navBar">
+            <a class="my-logo" href="#">My Logo</a>
+            <nav><img class="menuDropDown" src="images/Union.svg" alt="menuDropDown"></nav>
+        </header>
+        <section class="section1">
+            <div class="img-background">
+                <h1>I'm Mariane <br> Glad to see <br> you!</h1>
+                <p>I am a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, do not hestiate
+                    to contact me.</p>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul>
+                    <li><img class="section1-Icons" src="images/social-media-icons.svg" alt="social-Media-Icons"></li>
+                </ul>
+            </div>
+        </section>
+        <section class="main-Grid-Works">
+            <div class="works">
+                <div>
+                    <img class="snapshoot" src="images/Snapshoot-Portfolio1.svg" alt="Snapshoot-Image">
+                </div>
+                <h2 class="tonic">Tonic</h2>
+                <ul class="sub-Points">
+                    <li class="canopy">CANOPY</li>
+                    <li class="sub-Text">Back End Dev</li>
+                    <li class="sub-Text">2015</li>
+                </ul>
+                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="lang"> 
+                <ul class="languages">
+                    <li><button class="languages-List" type="submit">html</button></li>
+                   <li><button class="languages-List" type="submit">css</button></li>
+                    <li><button class="languages-List" type="submit">javascript</button></li>
+                </ul>
+            </div>
+                  
+                <button class="btn-Project" type="submit">See Project</button>
+
+            </div>
+            <div class="works">
+                <div>
+                    <img class="snapshoot" src="images/Snapshoot-Portfolio2.svg" alt="Snapshoot-Image">
+                </div>
+                <h2 class="tonic">Multi-Post Stories</h2>
+                <ul class="sub-Points">
+                    <li class="canopy">CANOPY</li>
+                    <li class="sub-Text">Back End Dev</li>
+                    <li class="sub-Text">2015</li>
+                </ul>
+                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li><button class="languages-List1" type="submit">html</button></li>
+                   <li><button class="languages-List2" type="submit">css</button></li>
+                    <li><button class="languages-List3" type="submit">javascript</button></li>
```

Comment:
- [ ] Kindly update the styling to look like the other li's, you missed this one you didn't add the right class names 😉 
---
Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![newbtn](https://user-images.githubusercontent.com/69638013/153738916-075a3955-ab36-4cc6-8cec-54dea2aa22d7.PNG)  | ![btnswrong](https://user-images.githubusercontent.com/69638013/153738942-c8901299-097e-4975-bc18-affa6f07500a.PNG)